### PR TITLE
Pp 65 update lessonscontroller back

### PR DESCRIPTION
**Pull request by mariyador**
### Description
Updates to createLesson.
Update deleteClass
add check for student age before applying for class

### CODE REVIEW

**Review by ofeterovskaya — APPROVED** [review]
Thank you Maria!

File: src/controllers/classesController.js [hunk 1]
```diff
@@ -294,6 +303,23 @@ const deleteClass = async (req, res) => {
 
 //apply for class
 
+// Function to calculate age from date of birth
+const calculateAge = (dateOfBirth) => {
```
Comment (AnnaPestova1) [inline src/controllers/classesController.js:307]: we have a very same function in utils/adultValidation. you can use it here

File: src/controllers/lessonsController.js [hunk 2]
```diff
@@ -133,6 +133,14 @@ const createLesson = async (req, res) => {
   const createdBy = req.user.userId;
   const userRole = req.user.role;
   const { studentId } = req.params;
+  const {
+    lessonTitle,
+    lessonDescription,
```
Comment (AnnaPestova1) [inline src/controllers/lessonsController.js:138]: later in this code you have repetitive. Please check which part of code you want o leave
image

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Maria! Great work. Check my comments. You definitely need to get rid of code duplicate const { lessonTitle, lessonDescription, lessonSchedule, type } = req.body

File: src/controllers/classesController.js (hunk 1 above)
Comment (mariyador) [reply src/controllers/classesController.js]: Done

File: src/controllers/lessonsController.js (hunk 2 above)
Comment (mariyador) [reply src/controllers/lessonsController.js:138]: Fixed

**Review by AnnaPestova1 — APPROVED** [review]
Thank you for changes

**Pull request merged**